be/src/io/cache/block_file_cache.cpp

```diff
@@ -1925,6 +2051,36 @@ std::map<std::string, double> BlockFileCache::get_stats() {
     return stats;
 }
 
+// for be UTs
+std::map<std::string, double> BlockFileCache::get_stats_unsafe() {
```

Comment:
private member functions can be access when UT, these code can be put to _test.cpp file